docs/templates/how-to-template.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Start with a verb ending in -ing
+content_type: how-to
+no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+beta: true # Labels the page as beta; adds a banner to the top of the page.
+disable_image_expand: true # Stops images from expanding in a modal on click. Sets it for the entire page.
+class: no-copy-code # Disables the copy code button for all code blocks on the page.
```

Comment:
We probably don't want to advertise these two. They're used infrequently and _usually_ shouldn't be used at all